images/dcos/bootstrap.sh

```diff
@@ -84,23 +79,19 @@ echo "* service proxy: $service_proxy"
 etcd_server_port=${ETCD_SERVER_PORT:-4001}
 etcd_server_peer_port=${ETCD_SERVER_PEER_PORT:-4002}
 
-ENABLE_ETCD_SERVER=${ENABLE_ETCD_SERVER:-false}
-if [ "${ENABLE_ETCD_SERVER}" != true ]; then
-  etcd_server_list=${ETCD_SERVER_LIST:-http://${service_proxy}:${etcd_server_port}}
-else
-  etcd_advertise_server_host=${ETCD_ADVERTISE_SERVER_HOST:-127.0.0.1}
-  etcd_server_host=${ETCD_SERVER_HOST:-127.0.0.1}
+ETCD_MESOS=${ETCD_MESOS:-disabled}
```

Comment:
we can add some short description here outlining the dependency.
unfortunately dependencies in DCOS are currently less than ideal.

On Wed, Oct 14, 2015 at 11:11 AM, James DeFelice <james@mesosphere.io>
wrote:

> it's a dependency
>
> On Wed, Oct 14, 2015 at 11:09 AM, Dr. Stefan Schimanski <
> notifications@github.com> wrote:
>
>> In images/dcos/bootstrap.sh
>> <https://github.com/mesosphere/kubernetes-mesos/pull/556#discussion_r42006139>
>> :
>>
>> > @@ -84,23 +79,19 @@ echo "* service proxy: $service_proxy"
>> >  etcd_server_port=${ETCD_SERVER_PORT:-4001}
>> >  etcd_server_peer_port=${ETCD_SERVER_PEER_PORT:-4002}
>> >
>> > -ENABLE_ETCD_SERVER=${ENABLE_ETCD_SERVER:-false}
>> > -if [ "${ENABLE_ETCD_SERVER}" != true ]; then
>> > -  etcd_server_list=${ETCD_SERVER_LIST:-http://${service_proxy}:${etcd_server_port}}
>> > -else
>> > -  etcd_advertise_server_host=${ETCD_ADVERTISE_SERVER_HOST:-127.0.0.1}
>> > -  etcd_server_host=${ETCD_SERVER_HOST:-127.0.0.1}
>> > +ETCD_MESOS=${ETCD_MESOS:-disabled}
>>
>> I understand what it does. But the name ETCD_MESOS does not suggest what
>> it does. What is etcd-mesos in the context of a mesos framework?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/mesosphere/kubernetes-mesos/pull/556/files#r42006139>
>> .
>>
>
>